src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        return new Ball(ColorSupplier.getRandomColor(), new Random().nextInt(100));

```

Comment:
Don't create several instances of the same class if you can use only one instance for your purposes. In this case, you're creating a new instance of `Random` every time the method is called, which is unnecessary. You should create a single instance of `Random` as a class field and reuse it.